linode/instance/schema_resource.go

```diff
@@ -208,6 +209,7 @@ var resourceSchema = map[string]*schema.Schema{
 		Optional:      true,
 		ForceNew:      true,
 		Sensitive:     true,
+		RequiredWith:  []string{"image"},
```

Comment:
It wasn't clear from the SDKv2 documentation whether the RequiredWith relationship was uni or bidirectional, but in practice it seems to work the way we want 🙂 

Another note, the validation error raised here isn't very intuitive:

```
all of `image,stackscript_data` must be specified
```

I wonder if it'd be worth implementing a CustomizeDiff validator to make this cleaner 🤔

Reply:
I'm getting a ton of testing failures due to this response. `instancedisk` runs fine but `instance` is full of failures.

Reply:
Weird, let me try it on my end

Reply:
Looks like RequiredWith only accepts absolute paths and cannot be used under nested lists according to the documentation. Since nested disks are deprecated anyway I just pushed up a PR to drop `RequiredWith` from them.

Reply:
After the changes i'm still getting the error on the following tests: (huge drop from before)
```
TestAccResourceInstance_typeChangeNoDisks
TestAccResourceInstance_powerStateNoImage
TestAccResourceInstance_typeChangeDiskExplicit
```

Reply:
Looks like some of the test templates are incorrect, let me update those real quick 🙂